backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
+(
+    test_methodology_id bigserial     not null primary key,
+    name                text          not null unique,
+    description         text          null,
+    confluence_link     varchar(1000) null
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id  bigserial                not null primary key,
+    project_id     bigint                   not null,
+    methodology_id bigint                   not null,
+    -- This is input from the user
+    name           text                     null,
+    meta           text                     null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name  text                     not null,
+    upload_date    timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
+            on delete cascade,
+
+    constraint has_methodology
+        foreign key (methodology_id) references test_methodology (test_methodology_id)
+            -- Methodology_id is immutable
+            on update no action
+            -- Prevent removing methodology if there is an experiment having such methodology
+            on delete no action
+);
+
+create table measurement
+(
+    measurement_id bigserial        not null primary key,
+    experiment_id  bigint           not null,
+    -- Either compound id or name
+    compound_id    text             not null,
+    concentration  double precision not null,
+    signal         double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete cascade
+            on update cascade
+);
+
+create table analysis_method
+(
+    analysis_method_id bigserial not null primary key,
+    description        text      null,
+    -- Analysis-specific parameters
+    parameters         json      null
+);
+
+create table analysis
+(
+    analysis_id        bigserial          not null primary key,
+    analysis_method_id bigint             not null,
+    experiment_id      bigint             not null,
+    compound_id        bigint             not null,
+    -- Maybe JSON better?
+    result             double precision[] not null,
```

Comment:
It's a list of numbers, right? What does it denote? In case of IC50 computation we will likely have two numbers: IC50 value and deviation. Is it correct that the list will have 2 numbers in this case? Do we have a list here as a generalization because other ways of analysis may produce different results?

If it's done for generalization, I think JSON is better. If it's done for a different reason, please explain it.

Reply:
>Do we have a list here as a generalization because other ways of analysis may produce different results?

yes and no. I wasn't sure that IC50 produces only 2 values, so I will change it to JSON